src/components/page/page.module.scss

```diff
@@ -42,6 +42,7 @@
     'grid-table-top' auto
     'grid-placeholder' auto
     'grid-media' auto
```

Comment:
probably to go along with our ditching `MediaTag` in the components, let's change these `grid-media`s to `grid-widget`s

Reply:
done and done